app/models/adventures/adventure_mission.rb

```diff
@@ -6,20 +6,30 @@ class AdventureMission < ApplicationRecord
   translates :title, :description
   globalize_accessors locales: [:en, :sv], attributes: [:title, :description]
 
-  attr_accessor :finished, :points
+  attr_accessor :finished, :points, :accepted, :require_acceptance
```

Comment:
Don't need the two added `attr_accessor`, actually something breaks if you include them.